phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css

```diff
@@ -122,6 +122,14 @@
     -fx-max-height: 2.5em;
 }
 
+#menuPane > .tab-header-area .tab:selected {
```

Comment:
Fine for me. What colors would you use for the dark one?

Or how about something like KDE:
![KDE like](https://cloud.githubusercontent.com/assets/3973260/22175838/8b396fb0-dffd-11e6-8354-8c91937f32ad.png)

Btw: Do you see the ugly frame around the left panel with the dark theme?